src/routes/menu-router.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+import menuController from '../controllers/menu-controller';
+
+const router = express.Router();
+
+router.route('/addMenu')
+  .post((req, res) => {
+    const buffer = [];
+    req.on('data', (chunk) => {
+      buffer.push(chunk);
+    }).on('end', () => {
+      const file = Buffer.concat(buffer);
+      menuController.addMenu(file)
+        .then((response) => {
+          res.status(200).send(response);
+        })
+        .catch((error) => {
+          res.status(406).send(error);
```

Comment:
Instead of 406 error try providing a more specific error code where possible. If it's generic error and you don't know where it came from use 500 status.